### ISSUE TITLE
How to clear cache after added node?

### ISSUE THREAD

**Issue by averichev** [Atlantic18/DoctrineExtensions#1871]
```php
$em->flush();
$em->clear();
```
This code not clear apcu cache. 

New node appears in result after this command running
```php
apcu_clear_cache();
```

```php
$repo = $this->getEntityManager()->getRepository('MyBundle:Tree');
$qb = $repo->getChildrenQueryBuilder($node, null, null, 'ASC', $withParent);
$qb->andWhere('node.active = TRUE');
$result = $qb->getQuery()->getResult(); 
// new node is missing in result before starting the command apcu_clear_cache();
```

Help please.

**Comment by AkenRoberts** [comment]
Hello, and thank you for your contribution.

In the interest of spring cleaning and package modernization, I'm going through and closing old issues that have had no recent activity and/or are no longer relevant.

If this problem is still relevant to you, please open a new issue with the appropriate information and steps to reproduce.

Thank you!
